tox.ini

```diff
@@ -3,16 +3,19 @@ recreate = true
 envlist = py-unittest
 indexserver =
     preinstall = https://pypi.python.org/simple
+requires=tox-conda
 
 [base]
 deps =
     :preinstall: -rrequirements.txt
     -rcompanion.txt
-    mkl
+    mkl;sys_platform!='darwin'
 
 [testenv]
 allowlist_externals = bash
-passenv:LAL_DATA_PATH
+passenv=LAL_DATA_PATH
+conda_deps=lalsuite
```

Comment:
Since we don't have a MacOSX runner with ARM, I guess can you confirm that this works there? Is it actually using the ARM versions?

Reply:
This does work on ARM and does run using ARM-compiled code ... Or at least it did work, there's currently an issue where using the newest openSSL version breaks things, pinning the version fixes it, but I think that's orthogonal to this PR.